python/cudf/groupby/groupby.py

```diff
@@ -208,7 +208,7 @@ def apply_multiindex_or_single_index(self, result):
                     levels.append([])
                     codes.append([])
                     names.append(by)
-                mi = MultiIndex(levels, codes)
+                mi = MultiIndex(levels, codes, source_data=result[self._by])
```

Comment:
This will initialize the levels and codes to empty lists instead of None.

Reply:
I simplified this way down, the new format makes all the empty lists redundant.